http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3103,16 +3103,12 @@ components:
       description: Creates a new runtime
       type: object
       required:
-        - region
         - machineSize
         - disk
       properties:
         labels:
           type: object
           description: The labels to be placed on the runtime. Of type Map[String,String]
-        region:
```

Comment:
Is Terra UI passing this today? May want to remove it.

Reply:
if UI is passing it, it will just not being used...but yea, we should have a follow up UI PR